sequencer/src/api/endpoints.rs

```diff
@@ -215,6 +215,28 @@ where
     Ok(api)
 }
 
+pub(super) fn node_state<S, ApiVer: StaticVersionType + 'static>(
+    _: ApiVer,
+) -> Result<Api<S, Error, ApiVer>>
+where
+    S: 'static + Send + Sync + ReadState,
+    S::State: Send + Sync + NodeStateDataSource,
+{
+    let toml = toml::from_str::<toml::Value>(include_str!("../../api/node_state.toml"))?;
+    let mut api = Api::<S, Error, ApiVer>::new(toml)?;
+    api.get("current_version", move |_, state| {
+        async move { Ok(state.get_node_state().await.current_version) }.boxed()
```

Comment:
Sorry, I should named it better but this current version in the node state is actually the base version

Reply:
I think it should be renamed to base version to avoid confusion. If we really need the current version of the node then it should be stored in validated state and retrieved from it

Reply:
Oh yea I misunderstood! thanks for the clarification!

Reply:
If I understand, we first need to add the version field to validated_state, since it is not currently present. Then we need to pass validated_state into `ApiState` somehow. Not quite as easy as I had hoped. Maybe we should just go w/ chain_config now, since we can tell by the values in chain_config what version we are on.

Reply:
Nevermind, we have the same problem with chain_config :face_exhaling: